packages/tools/locales/fr_FR.po

```diff
@@ -75,7 +78,7 @@ msgstr ""
 #: packages/app-desktop/plugins/GotoAnything.js:456
 #: packages/app-desktop/gui/KeymapConfig/utils/getLabel.js:20
 msgid "Goto Anything..."
-msgstr "Navigation rapide…"
+msgstr "aller n'importe ou... "
```

Comment:
Is there a reason why there's a space after `...`?
I've noticed this in many places, but I won't mark every single one of them.

Sometimes there are even 2 spaces with a sentence. I will mark the first occurrence.